Packs/MongoDB/Integrations/MongoDB/MongoDB_test.py

```diff
@@ -33,3 +36,74 @@ def test_convert_id_to_object_id(func_input, expected):
 @pytest.mark.parametrize("func_input, expected", object_to_id)
 def test_convert_object_id_to_str(func_input, expected):
     assert expected == convert_object_id_to_str(func_input)
+
+
+def test_normalize_id():
+    res = Client.normalize_id({'_id': ObjectId('5e4412f230c5b8f63a7356ba')})
+    assert '5e4412f230c5b8f63a7356ba' == res['_id']
+
+
+class TestConvertStrToDatetime:
+    dict_inputs = [
+        {"testing": 123, "time": "ISODate('2020-06-12T08:23:07.000Z')"},
+        pytest.param(
+            {"testing": 123, "time": "ISODate('2018-06-12T08:23:07.000')"},
+            marks=pytest.mark.xfail),
+    ]
+
+    @pytest.mark.parametrize("func_input", dict_inputs)
+    def test_convert_str_to_datetime(self, func_input):
+        res = convert_str_to_datetime(func_input)
+        assert isinstance(res['time'], datetime)
+
+    def test_convert_str_to_datetime_no_datetime_obj(self):
+        inputs = {1: 2}
+        res = convert_str_to_datetime(inputs)
+        assert isinstance(res[1], int)
+
+
+class TestDatetimeToStr:
```

Comment:
add test for object of type `Cursor`

Reply:
There's a problem creating this object outside mongodb. it anyway reacts like a list.